### PR TITLE
Add LogSize to container inspect

### PR DESCRIPTION
**Pull request by rhatdan**
Other log options are available so we need to add ability to look
up LogSize.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
This PR will help to clarify that https://github.com/containers/podman/pull/8646 is not needed.

File: libpod/define/container_inspect.go [hunk 1]
```diff
@@ -622,6 +622,7 @@ type InspectContainerData struct {
 	OCIRuntime      string                      `json:"OCIRuntime,omitempty"`
 	LogPath         string                      `json:"LogPath"`
 	LogTag          string                      `json:"LogTag"`
+	LogSize         string                      `json:"LogSize"`
```
Comment (mheon) [inline libpod/define/container_inspect.go:625]: This should actually go in `InspectLogConfig` - https://github.com/containers/podman/blob/15ac7c5313d58c12f46dfa7eaa6ade05aca3df59/libpod/define/container_inspect.go#L84-L89

Comment (mheon) [reply libpod/define/container_inspect.go:625]: I don't know if it should be part of Config or a separate field, though. Either works for me.

Comment (rhatdan) [reply libpod/define/container_inspect.go:625]: You want it in a different place the LogPath and LogTag?

Comment (mheon) [reply libpod/define/container_inspect.go:625]: Yes. LogConfig is the more correct place (it's where `docker inspect` sticks all their log settings). We should probably move the other options in there as well.

Comment (TomSweeneyRedHat) [reply libpod/define/container_inspect.go]: @rhatdan, I thought @mheon wanted it moved here?  https://github.com/containers/podman/blob/5cf7aa65fb4786ee3dad191a725f6c37fae83bc3/pkg/specgen/specgen.go#L14
Or am I mistaken?

Comment (rhatdan) [reply libpod/define/container_inspect.go]: Yes you are correct. I missed that.

**Comment by zhangguanzhang** [comment]
LGTM

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm

**Review by vrothberg — COMMENTED** [review]
/lgtm

**Pull request merged**